include/bouncer.h

```diff
@@ -326,7 +327,7 @@ struct PgPool {
 	struct List map_head;			/* entry in user->pool_list */
 
 	PgDatabase *db;			/* corresponding database */
-	PgUser *user;			/* user logged in as, this field is NULL for peer pools */
+	PgAuthInfo *user;			/* user logged in as, this field is NULL for peer pools */
```

Comment:
Given GlobalUser as a type, I am tempted to suggest this field should be called auth_info or credentials instead of user, to reduce confusion. (And yes, this implies a fair amount of substitutions elsewhere.)

In general, how do you feel about never using "user" for pgAuthInfo data, and keeping that reserved for GlobalUser types?

Reply:
Yeah, it's definitely something I think we should be consistent about and was originally trying to do what you are suggesting. But when I got to the `auth_user` field, I didn't really like the resulting `auth_user_auth_info` name.

So I held off on changing everything, because maybe it fits better to rename `PgAuthInfo` to `PgUser`

Reply:
What about s/PgAuthInfo/PgCredentials/, so that you end up with `auth_user_credentials` or the like?

Reply:
I like that suggestion. I'll try that out tomorrow.

Reply:
Okay, I finally found the time to do this. Could you take another look? I'm pretty happy with the result.